pkgs/applications/office/tusk/default.nix

```diff
@@ -0,0 +1,33 @@
+{ appimageTools, fetchurl, lib, gsettings-desktop-schemas, gtk3 }:
+
+let
+  pname = "tusk";
+  version = "v0.23.0";
+in appimageTools.wrapType2 rec {
+  name = "${pname}-${version}";
+  src = fetchurl {
+    url = "https://github.com/klaussinani/tusk/releases/download/v0.23.0/tusk-0.23.0-x86_64.AppImage";
```

Comment:
Please take advantage of pname and version variables.

Reply:
good call